src/services/resolve/services/scopeResolverService.js

```diff
@@ -12,6 +12,30 @@ const getDataEntry = ({ type, id, name, authorities = ['can-read'], attributes =
 	},
 });
 
+const getPermissionsForScope = (
```

Comment:
Can you please add some tests?
https://github.com/hpi-schul-cloud/schulcloud-server/tree/main/test/services/resolve

https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/test/services/helpers/testObjects.js#L146 (look to examples that us it)